src/Module/Admin/Users.php

```diff
@@ -157,15 +157,15 @@ public static function content(array $parameters = [])
 		$pager = new Pager(DI::l10n(), DI::args()->getQueryString(), 100);
 
 		$valid_orders = [
-			'contact.name',
-			'user.email',
-			'user.register_date',
-			'user.login_date',
-			'lastitem_date',
-			'user.page-flags'
+			'name',
+			'email',
+			'register_date',
+			'login_date',
+			'last-item',
+			'page-flags'
 		];
 
-		$order = 'contact.name';
+		$order = 'name';
 		$order_direction = '+';
```

Comment:
The variable is used at other places in this function as well. Especially with a PR that large I prefer to not apply more changes that needed. Otherwise this PR will be some never ending story.